src/features/dashboard/NFTCard.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
```

Comment:
@CrazyOptimist as far as I know, NFTCard has been created by Bulat. You should use it instead. Here you go: https://github.com/pastelnetwork/pastel-electron-wallet/blob/feature/nft-market-40026661/src/common/components/NFTCard/NFTCard.tsx

Reply:
You are right, I will replace it after the PR would be merged into new_development, you can think that it's just a placeholder for now.

Reply:
@CrazyOptimist no, PR [#112](https://github.com/pastelnetwork/pastel-electron-wallet/pull/112) is ready for review. You can merge `feature/nft-market-4002661` into your branch or you should wait until the PR #112 is merged.

Reply:
> 
> 
> @CrazyOptimist remove the line
> ![image](https://user-images.githubusercontent.com/71397124/120261200-68c29480-c265-11eb-9e4f-d36ea194e36c.png)

It's horizontal scroll which appears only when the screen width is between1100px - 1200px

Reply:
Okay, I will check the NFT card

Reply:
Adopted the NFT card.